app/_components/module-launch-pad/ModuleLaunchPad.tsx

```diff
@@ -1,133 +1,134 @@
 import useColorMode from '@/app/_hooks/useColorMode';
-
-interface Module {
-  title: string;
-  url: string;
-  icon: React.ReactNode;
-  description: string;
-}
-
-const modules: Module[] = [
-  {
-    url: '/systems',
-    icon: <Systems />,
-    title: 'Systems',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/bodies',
-    icon: <Bodies />,
-    title: 'Bodies',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/stations',
-    icon: <Stations />,
-    title: 'Stations',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/attractions',
-    icon: <Attractions />,
-    title: 'Attractions',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/pois',
-    icon: <POIs />,
-    title: 'POIs',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/factions',
-    icon: <Factions />,
-    title: 'Factions',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/shipyard',
-    icon: <Shipyard />,
-    title: 'Shipyard',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/trade/single',
-    icon: <SingleTradeRoute />,
-    title: 'Single Trade Route',
-    description:
-      'Trade from A to B and find the best profit route using multiple filter options',
-  },
-  {
-    url: '/trade/multi',
-    icon: <MultiHopTradeRoute />,
-    title: 'Multi-Hop Trade Route',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-  {
-    url: '/trade/loop',
-    icon: <LoopTradeRoute />,
-    title: 'Loop Trade Route',
-    description:
-      'Browse the universe! Jump to any system or search by many properties',
-  },
-];
+import ModuleProps, { Module } from './moduleProps';
 
 const ModuleLaunchPad = () => {
   const { isDark } = useColorMode();
 
-  return (
-    <SimpleGrid minChildWidth="220px" spacing="50px">
-      {modules.map((module) => (
-        <LinkBox
-          key={module.title}
-          borderWidth="1px"
-          borderRadius="9px"
-          borderColor={isDark ? 'dark.text' : 'light.text'}
-          p="12px"
-          position="relative"
-        >
-          <Heading
-            as="h2"
-            size="sm"
-            display="flex"
-            gap="8px"
-            alignItems="center"
-            mb="10px"
+  // local functions
+  const selectColor = (element: String) => {
+    if (element === 'text') return isDark ? 'dark.text' : 'light.text';
+    if (element === 'box') return isDark ? 'dark.box' : 'light.box';
+    return isDark ? 'dark.background' : 'light.background';
+  };
```

Comment:
Should this be a common function. Looks like this will help control box theming across the site. Not critical to sort this now, we can revisit when we need to repeat this behavior elsewhere.

Reply:
Agreed. added a helper function for this [#3776ec1](https://github.com/ed-pilots-network/frontend/pull/36/commits/3776ec1e5dfebdde5b9a7840c84cde5c91318e11)